src/core/memory.cc

```diff
@@ -1798,6 +1799,8 @@ void maybe_dump_memory_diagnostics(size_t size) {
 }
 
 void on_allocation_failure(size_t size) {
+    alloc_stats::increment(alloc_stats::types::failed_allocs);
+
```

Comment:
Should be folded into the commit that adds the increment in the first place.

Reply:
Done.